projects/web/webpack.config.js

```diff
@@ -1,6 +1,9 @@
 var webpack = require("webpack");
 
 module.exports = (env) => {
+  console.log("LALALALALALALALALALALALALALALA");
```

Comment:
was this meant to be checked in?

Reply:
I don't know what you are talking about :see_no_evil: